packages/integrations/node/src/index.ts

```diff
@@ -24,6 +24,16 @@ export function getAdapter(options: Options): AstroAdapter {
 	};
 }
 
+// TODO: remove once we don't use a TLA anymore
+async function shouldExternalizeAstroEnvSetup() {
+	try {
+		await import('astro/env/setup');
+		return false;
+	} catch {
+		return true;
```

Comment:
@bluwy I think we need to externalize only if it can't be imported?

Reply:
Yup 👍